CHANGELOG.md

```diff
@@ -52,9 +52,11 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 
 * (sims) [#21906](https://github.com/cosmos/cosmos-sdk/pull/21906) Skip sims test when running dry on validators
 * (cli) [#21919](https://github.com/cosmos/cosmos-sdk/pull/21919) Query address-by-acc-num by account_id instead of id.
+* (bank) [#22543](https://github.com/cosmos/cosmos-sdk/pull/22543) Fix `DenomMetadata` rpc allow value with slashes
```

Comment:
Could you delete this changelog entry?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/22543/commits/e97e236f10b31e3383d157cb83b7bc8c15343c62